src/services/sync/strategies/CSVSyncer.js

```diff
@@ -8,13 +8,11 @@ const Syncer = require('./Syncer');
  * @implements {Syncer}
  */
 class CSVSyncer extends Syncer {
-	constructor(app, stats = {}, school, role = 'student', sendEmails, csvData, params) {
+	constructor(app, stats = {}, options = {}, requestParams = {}) {
```

Comment:
what are possible options? Which is the reason, to replace the parameters with an option object?

Reply:
Parameters are set via the `params` method. https://github.com/schul-cloud/schulcloud-server/pull/679/files/3a96dff674a92c77f936b96f6a56d6b5ce68d05b#diff-53d84f5fa2f6f6d0b028d6a12bc9adaeR47